openapi/openapi.yml

```diff
@@ -268,120 +268,142 @@ components:
+          example: "1234123asdf4tYadsfQ,xY-01BU12"
+        impression:
           type: string
-          example: arstechnica.com
-        image:
+          example: "a0c3943asdf4tYadsf300,xY-01BU9aadc"
+        delete:
           type: string
-          example: "https://img-getpocket.cdn.mozilla.net/direct?url=https%3A%2F%2Fcdn.arstechnica.net%2Fwp-content%2Fuploads%2F2018%2F12%2FGettyImages-898172236-640x427.jpg&resize=w450"
-        raw_image_src:
+          example: "fdea123asdf4tYadsf1000,xY-01BUa654"
+        save:
           type: string
-          example: "https://cdn.arstechnica.net/wp-content/uploads/2018/12/GettyImages-898172236-640x427.jpg"
-        published_timestamp:
+          example: "4567123asdf4tYadsfQcda,xY-01BU123"
+
+    Caps:
+      type: object
+      additionalProperties: false
+      required:
+        - lifetime
+        - flight
+        - campaign
+      properties:
+        lifetime:
           type: integer
-          description: Unix timestamp of when this item was published
-          example: 1543471200
-        item_score:
-          type: number
-          example: 0.98
-        parameter_set:
+          example: 50
+        flight:
+          type: object
+          additionalProperties: false
+          required:
+            - count
+            - period
+          properties:
+            count:
+              type: integer
+              example: 10
+            period:
+              type: integer
+              description: Period in seconds
+              example: 86400
+        campaign:
+          type: object
+          additionalProperties: false
+          required:
+            - count
+            - period
+          properties:
+            count:
+              type: integer
+              example: 10
+            period:
+              type: integer
+              description: Period in seconds
+              example: 86400
+
+    SpocFeedItem:
+      type: object
+      additionalProperties: false
+      properties:
+        campaign_id:
+          type: integer
+          example: 784
+        caps:
+          type: object
+          $ref: '#/components/schemas/Caps'
+        collection_title:
           type: string
-          example: default
+          description: Shared title if all ads are one collection
+        context:
+          type: string
+          description: Deprecated. Use sponsor field instead.
+          example: Sponsored by NextAdvisor
+        cta:
+          type: string
+          description: Text to display on CTA button
+          example: Learn more
+        domain:
+          type: string
+          example: arstechnica.com
         domain_affinities:
           type: object
           additionalProperties:
             type: number
           example:
             vanguard.com: 0.9956
             wealthsimple.com: 0.9193
-
-    SpocFeedItem:
-      type: object
-      allOf:
-        - $ref: '#/components/schemas/FeedItem'
-        - type: object
```

Comment:
the "allOf" with a ref and object means both have to match, but it seemed weird to split the object in half (there isn't a secondary type in the api that shares a portion of the fields) so I combined them into a single item.